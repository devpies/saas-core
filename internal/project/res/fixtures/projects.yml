- project_id: 96c3424e-17cf-4bd2-916e-1ec2ddc979a5
  tenant_id: f24d653a-f465-11ec-bfa4-26b2e5d16858
  name: Apple
  prefix: APP-
  description: The Apple project
  user_id: 0ef64d03-8a91-4513-907c-dd1fcfcfeb46
  active: true
  public: false
  column_order: RAW='{"column-1", "column-2", "column-3", "column-4"}'
  updated_at: 2022-07-15T09:08:27Z
  created_at: 2022-07-15T09:08:27Z

- project_id: f8a6daf8-7239-47c3-a4e7-74d46439c7e5
  tenant_id: f24d653a-f465-11ec-bfa4-26b2e5d16858
  name: Lime
  prefix: LIM-
  description: ""
  user_id: 0ef64d03-8a91-4513-907c-dd1fcfcfeb46
  active: true
  public: false
  column_order: RAW='{"column-1", "column-2", "column-3", "column-4"}'
  updated_at: 2022-07-15T09:08:27Z
  created_at: 2022-07-15T09:08:27Z
