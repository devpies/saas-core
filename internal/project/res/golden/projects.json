[
  {
    "id": "96c3424e-17cf-4bd2-916e-1ec2ddc979a5",
    "tenantId": "f24d653a-f465-11ec-bfa4-26b2e5d16858",
    "name": "Apple",
    "prefix": "APP-",
    "description": "The Apple project",
    "userId": "0ef64d03-8a91-4513-907c-dd1fcfcfeb46",
    "active": true,
    "public": false,
    "columnOrder": [
      "column-1",
      "column-2",
      "column-3",
      "column-4"
    ],
    "updatedAt": "2022-07-15T09:08:27Z",
    "createdAt": "2022-07-15T09:08:27Z"
  },
  {
    "id": "f8a6daf8-7239-47c3-a4e7-74d46439c7e5",
    "tenantId": "f24d653a-f465-11ec-bfa4-26b2e5d16858",
    "name": "Lime",
    "prefix": "LIM-",
    "description": "",
    "userId": "0ef64d03-8a91-4513-907c-dd1fcfcfeb46",
    "active": true,
    "public": false,
    "columnOrder": [
      "column-1",
      "column-2",
      "column-3",
      "column-4"
    ],
    "updatedAt": "2022-07-15T09:08:27Z",
    "createdAt": "2022-07-15T09:08:27Z"
  }
]
